internal/types/document_validation.go

```diff
@@ -38,9 +39,9 @@ func (e *ValidationError) Error() string {
 // ValidateData checks if the document represents a valid "data document".
 // If the document is not valid it returns *ValidationError.
 func (d *Document) ValidateData() error {
-	// The following block should be used to checks that keys are valid.
-	// All further key related validation rules should be added here.
```

Comment:
@rumyantseva is there any background behind making this loop only for keys? Can I leave the change or should I make another loop for values?

Reply:
@noisersup up to you. Please take a looks at https://github.com/FerretDB/FerretDB/pull/1293, there won't be separate way to store values soon, so ost likely, both keys and values will be in the same loop.